receiver/pluginreceiver/rendered_config_test.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I think my IDE may have replaced tabs with spaces here. I'll remove this file from the PR.